DOCS.md

```diff
@@ -1,19 +1,52 @@
-Use the PyPI plugin to deploy a Python package to a PyPI server.
+Use the PyPi plugin to deploy a Python package to a PyPI server. You can
+override the default configuration with the following parameters:
 
-* `repository` - The repository name (optional)
-* `username` - The username to login with (optional)
-* `password` - A password to login with (optional)
-* `distributions` - A list of distribution types to deploy (optional)
+* `username` - Username to login
+* `password` - Password to login
 
-The following is an example configuration for your .drone.yml:
+## Example
+
+The following is a sample configuration in your .drone.yml file:
+
+```yaml
+publish:
+  pypi:
+    username: guido
+    password: secret
+```
+
+### Source dist
+
+In some cases you may want to upload a source distribution or a wheel to some
+PyPi server, for this use case we expose the following additional parameter:
+
+* `distributions` - List of distribution types
+
+Example configuration that uploads to a private PyPi server:
```

Comment:
This statement appears to be false. The default is to upload to the official package index.